package/environments/production.js

```diff
@@ -10,11 +10,16 @@ module.exports = class extends Environment {
 
     this.plugins.append('UglifyJs', new webpack.optimize.UglifyJsPlugin({
       sourceMap: true,
+      mangle: {
+        safari10: true
+      },
       compress: {
-        warnings: false
+        warnings: false,
+        comparisons: false
       },
       output: {
-        comments: false
+        comments: false,
+        ascii_only: true
```

Comment:
What do these changes do?

Reply:
https://github.com/mishoo/UglifyJS2/tree/harmony#output-options

Reply:
I mean, why are you adding/changing these options? Do they resolve any known issues?

Reply:
No, we didn't received any but just standardising the options for now (recommended ones).

Reply:
Recommended where?

Reply:
Just noticed a lot of bugs here related to emojis and regex and it seemed like a good option to be on since emojis are everywhere :): https://github.com/mishoo/UglifyJS2/search?p=1&q=ascii_only&type=Issues&utf8=%E2%9C%93 

They haven't turned this on by default because build css size increases or something.

Reply:
If you are not happy I can turn it off and wait until an issue is reported. Since the standard config is now hidden, thought it would make sense to add options that commonly causes issues.

Reply:
> Just noticed a lot of bugs here related to emojis and regex and it seemed like a good option to be on since emojis are everywhere :)

👍

> They haven't turned this on by default because build css size increases

CSS size❓

> If you are not happy I can turn it off and wait until an issue is reported

I'm ok with it as long as it doesn't bloat *JS* file sizes significantly

Reply:
Just tested, turning it on doesn't increase the build size (all hello examples).

Reply:
Oops typo there :) (ignore css)